src/main/java/org/ua/javaPro/berezhnoy/hillelHomeWork13/optionallyHW/ArrayInitializerForkJoin.java

```diff
@@ -0,0 +1,33 @@
+package org.ua.javaPro.berezhnoy.hillelHomeWork13.optionallyHW;
+
+import java.util.concurrent.RecursiveAction;
+
+public class ArrayInitializerForkJoin extends RecursiveAction {
+
+    private static final int hold = 20;
```

Comment:
```suggestion
    private static final int HOLD = 20;
```
константи зазвичай з великих літер.